test/test_common.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import os
+from pathlib import Path
+import sys
+
+BASE_DIR = Path(__file__).parent.parent
+PKG_DIR = BASE_DIR / 'packages'
+
+# TODO: remove once we have a proper Python package for common functions
+sys.path.append(str(BASE_DIR / 'tools'))
+import common  # noqa
+
+
+def registered_packages():
+    """Returns a list of registred package names"""
+    packages = [name for name in os.listdir(PKG_DIR)
+                if (PKG_DIR / name).is_dir()]
+    return packages
+
+
+def registered_packages_meta():
+    """Returns a dictionary with the contents of `meta.yaml`
+    for each registed package
+    """
+    packages = registered_packages
+    return {name: common.parse_package(PKG_DIR / name / 'meta.yaml')
+            for name in packages}
+
+
+UNSUPPORTED_PACKAGES = {'ChromeWrapper': ['pandas'],
+                        'FirefoxWrapper': []}
```

Comment:
I think this is fine for now.  If this grows too much or when we make separate package building a reality, we might want to move this so the package `meta.yaml` file specifies this.

Reply:
Definitely, adding it in `meta.yaml`: e.g. as a selector after the import, same as [conda defines OS specific commands](https://github.com/conda-forge/scikit-learn-feedstock/blob/f6bcde64895a985cd8a4383681b253998ec600d2/recipe/meta.yaml#L50-L51) would indeed be cleaner.